.github/workflows/release.yaml

```diff
@@ -27,17 +28,30 @@ jobs:
 
     - name: Docker Login
       uses: docker/login-action@v1
-      if: startsWith(github.ref, 'refs/tags')
```

Comment:
Looks like the release action is failing when attempting to log into the docker using the secret credentials. It seems like this is the correct behavior: https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow. I might be worth adding this conditional check back in place.